### PR TITLE
Add Last Elevate Event Played field

### PR DESCRIPTION
**Pull request by npsp-reedestockton**
# Critical Changes

# Changes
 - We added a new field, Last Elevate Event Played, to the Recurring Donation object. This field supports Elevate integration with NPSP. It's a system field so you don't need to add to your page layouts. We included Read access on the Standard and System Administrator profiles.

# Issues Closed

# Community Ideas Delivered

# New Metadata
- Custom field:
    - npe03__Recurring_Donation__c.LastElevateEventPlayed__c

# Deleted Metadata

### CODE REVIEW

**Comment by npsp-reedestockton** [comment]
@RadSFDeveloper @andrewyu-salesforce @sam-knox This is ready for review.

**Comment by npsp-reedestockton** [comment]
Tracking [W-8246650](https://gus.lightning.force.com/lightning/_classic/%2Fa07B0000008h6ESIAY)

**Review by andrewyu-salesforce — APPROVED** [review]
looks great

**Review by RadSFDeveloper — APPROVED** [review]
Awesome work, thank you @npsp-reedestockton !

File: src/objects/npe03__Recurring_Donation__c.object [hunk 1]
```diff
@@ -336,6 +336,18 @@
         <type>Number</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>LastElevateEventPlayed__c</fullName>
```
Comment (RadSFDeveloper) [inline src/objects/npe03__Recurring_Donation__c.object:340]: @callen-sfdo , do you please mind double checking the new field on RD that will contain an Elevate Event Id? Thanks.

Comment (sam-knox) [reply src/objects/npe03__Recurring_Donation__c.object:340]: @RadSFDeveloper - I assume we don't need to tell admins to put this field on their page layouts right?

**Review by sam-knox — APPROVED** [review]

File: src/objects/npe03__Recurring_Donation__c.object (hunk 1 above)
Comment (RadSFDeveloper) [reply src/objects/npe03__Recurring_Donation__c.object:340]: That is correct @sam-knox , the field is needed for the RD integration framework for an org connected to Elevate. It should not be added to any page layout.

Comment (sam-knox) [reply src/objects/npe03__Recurring_Donation__c.object:340]: @RadSFDeveloper - As part of documentation, do we need to tell admins to add Read and Edit access to this field for their Elevate integration user, or is this going to be added to the Elevate permission sets?

**Review by sam-knox — APPROVED** [review]
Updated the Changes section.

File: src/objects/npe03__Recurring_Donation__c.object (hunk 1 above)
Comment (RadSFDeveloper) [reply src/objects/npe03__Recurring_Donation__c.object:340]: The field is relevant for the integration user, so for the regular user, the check will be in trigger that will not check field permission, so there is no need for adding permissions. When the RD MetaDeploy is run, the field will be added to the system admin and standard user profile. So, for existing Elevate customers, we can specify that they add read/edit field permission to the field.

**Pull request merged**